config/services/buckets-mdapi/service.json

```diff
@@ -4,6 +4,6 @@
 		"ram": 256
 	},
 	"metadata": {
-		"MORAY_MAX_PG_CONNS": 16
+		"BUCKETS_MDAPI_CUEBALL_MAX_CONNECTIONS": 16
```

Comment:
You want to reduce the cueball conns to 16 for *coal*?
I think this'll have the effect of setting it to 16 for *production* size as well, down from the hardcoded default of 64 in the sapi template:

```
[cueball]
{{#BUCKETS_MDAPI_CUEBALL_MAX_CONNECTIONS}}
max_connections = {{BUCKETS_MDAPI_CUEBALL_MAX_CONNECTIONS}}
{{/BUCKETS_MDAPI_CUEBALL_MAX_CONNECTIONS}}
{{^BUCKETS_MDAPI_CUEBALL_MAX_CONNECTIONS}}
max_connections = 64
{{/BUCKETS_MDAPI_CUEBALL_MAX_CONNECTIONS}}
```

Reply:
My bad, yes it does change max connections to 16 as well for prod. Checking the config in the ticket again, we can just make it 32 for coal and keep it at 64 for other sizes.